api/src/api/session.js

```diff
@@ -30,6 +30,46 @@ router.get(
   }),
 );
 
+//get all displayed sessions on the home page
+router.get(
+  '/displayed',
+  errorWrap(async (req, res) => {
+    let activeSessions = await Session.find({ active: 'true' });
+    if (activeSessions.length >= 10) {
+      res.status(200).json({
+        message: `Successfully retrieved all displayed sessions on the home page.`,
+        success: true,
+        result: activeSessions,
+      });
+      return;
+    }
+
+    const inactiveSessions = await Session.find({ active: 'false' }).sort({
+      startTime: 'asc',
+    });
+
+    if (activeSessions.length + inactiveSessions.length < 10) {
+      let allSessions = activeSessions.concat(inactiveSessions);
+      res.status(200).json({
+        message: `Successfully retrieved all displayed sessions on the home page.`,
+        success: true,
+        result: allSessions,
+      });
+      return;
+    }
+
```

Comment:
Add comments throughout this endpoint to explain what each section is doing since this is a more complicated endpoint

Reply:
Gotcha